README.md

```diff
@@ -152,6 +110,46 @@ const plugin = createPlugin({
 
 * `[captureIntrospectionQueries = false]` Enable capture of timings for an [IntrospectionQuery](https://graphql.org/graphql-js/utilities/#introspectionquery).
 
+### Apollo Federation Support
+
+The New Relic plugin will work with an Apollo Federated Server out of the box. Just pass the plugin to the Federated Gateway Apollo Server. With the federated server you will get:
+* Distributed Tracing support
+* Transaction naming
+* Operation naming
+* Metrics
+
+Resolver spans are not supported for the Federated Gateway Server. We strongly recommend adding the plugin to your sub-graph servers which will generate all the data you get with the Federated Server as well as resolver spans.
+
+```js
+// Federated Gateway Server index.js
+const plugin = require('@newrelic/apollo-server-plugin')
+
+const gateway = new ApolloGateway({
+  serviceList: [
+    { name: 'server1', url: 'server1' },
+    { name: 'server2', url: 'server2' }
+  ]
+});
+
+const server = new ApolloServer({
+  gateway,
+  plugins: [ plugin ]
+});
+```
+
+```js
+// Sub-Graph server index.js
+const { buildFederatedSchema } = require('@apollo/federation');
+const plugin = require('@newrelic/apollo-server-plugin')
+
+const server = new ApolloServer({
+  schema: buildFederatedSchema([{ typeDefs, resolvers }]),
+  plugins: [ plugin ]
+});
+```
+
+`@apollo/federation` and `@apollo/gateway` are currently 0.x versions and may break with future versions prior to 1.x release. from Apollo.
```

Comment:
Looks like a rogue `.` made it into the end of the sentence. May also be worth highlight it slightly more? Doesn't matter that much but would remove the period.

```suggestion
**NOTE:** `@apollo/federation` and `@apollo/gateway` are currently 0.x versions and may break with future versions prior to 1.x release from Apollo.
```